src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.ValidateToken.Internal.cs

```diff
@@ -48,6 +48,19 @@ internal async Task<ValidationResult<ValidatedToken>> ValidateTokenAsync(
                 return conditionsResult.UnwrapError().AddStackFrame(StackFrames.AssertionConditionsValidationFailed);
             }
 
+            var issuerValidationResult = await validationParameters.IssuerValidatorAsync(
```

Comment:
This is a not yet released class right? So this *isn't* a behavorial breaking change?

Reply:
That is correct. This is all internal and work in progress. There's a few more PRs coming adding functionality to this method.

Reply:
Thank you, makes sense